master/buildbot/db/logs.py

```diff
@@ -155,7 +197,7 @@ def _splitBigChunk(self, content, logid):
         truncline = content[:self.MAX_CHUNK_SIZE]
         while truncline:
             try:
-                truncline.decode('utf-8')
+                truncline.decode(self.master.config.logEncoding)
```

Comment:
no, we should remove logEncoding, and put everything in utf-8.

All data in db shall be utf-8